### ISSUE TITLE
Examples don't run on Python 3.6.5

### ISSUE THREAD

**Issue by davidssmith** [jyhmiinlin/pynufft#13]
I get this error when trying to run the 1D example:
```
Traceback (most recent call last):
  File "script_1D.py", line 15, in <module>
    NufftObj.plan(om, Nd, Kd, Jd)
  File "/usr/local/lib/python3.6/site-packages/pynufft/nufft.py", line 75, in plan
    self.st = helper.plan(om, Nd, Kd, Jd)
  File "/usr/local/lib/python3.6/site-packages/pynufft/src/_helper/helper.py", line 149, in plan
    Nd[dimid], Jd[dimid], Kd[dimid])
  File "/usr/local/lib/python3.6/site-packages/pynufft/src/_helper/helper.py", line 433, in nufft_alpha_kb_fit
    coef = numpy.linalg.lstsq(numpy.nan_to_num(X), numpy.nan_to_num(sn_kaiser), rcond = None)[0]
  File "/usr/local/lib/python3.6/site-packages/numpy/linalg/linalg.py", line 1935, in lstsq
    0, work, -1, rwork, iwork, 0)
TypeError: must be real number, not NoneType
```

I get a similar error when running the 2D example as well.

**Comment by jyhmiinlin** [comment]
Confirmed. Thanks!

Fixed in 0.3.3.8: rcond = -1 in numpy.linalg.lstsq to be backward compatible with numpy-1.13.1 and Anaconda 3.6.5. So a change of rcond from None to -1 seems to work, for now.

It appears None is only accepted in numpy 1.14. (https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.linalg.lstsq.html) "To silence the warning and use the new default, use rcond=None, to keep using the old behavior, use rcond=-1. "

J

**Comment by jyhmiinlin** [comment]
This bug has been fixed for Python 3.6.5. Closed.
